UI/data/locale/en-US.ini

```diff
@@ -917,10 +917,8 @@ Basic.Settings.Stream.TTVAddon.BTTV="BetterTTV"
 Basic.Settings.Stream.TTVAddon.FFZ="FrankerFaceZ"
 Basic.Settings.Stream.TTVAddon.Both="BetterTTV and FrankerFaceZ"
 Basic.Settings.Stream.MissingSettingAlert="Missing Stream Setup"
+Basic.Settings.Stream.MissingSetting="The service is misconfigured. \n\nOpen settings to and check the 'Stream' tab."
 Basic.Settings.Stream.StreamSettingsWarning="Open Settings"
-Basic.Settings.Stream.MissingUrlAndApiKey="URL and Stream Key are missing.\n\nOpen settings to enter the URL and Stream Key in the 'stream' tab."
-Basic.Settings.Stream.MissingUrl="Stream URL is missing.\n\nOpen settings to enter the URL in the 'Stream' tab."
-Basic.Settings.Stream.MissingStreamKey="Stream key is missing.\n\nOpen settings to enter the stream key in the 'Stream' tab."
```

Comment:
> Open settings to and check
A bit of an English error here. I'm guessing this was originally copied/pasted.

But this error reporting could probably use some reworking. I'm not entirely sure what to suggest here though. This all-encompassing error message is actually *less* helpful than the old errors. At least with the old errors the users knew what to change or knew what to do. With this error message, the users have a more vague message that sounds like "your settings are bad, try again". What settings are bad? The users probably need to know that.

Reply:
In RFC 39, we no longer to be able to guess what is wrong since services will no longer be only first party.
But I'll move this change to RFC 39 since we can add warnings and errors in the service property view.